layouts/partials/footer.html

```diff
@@ -6,66 +6,50 @@
     </div>
     <footer>
         <div class="footer">
+            <figure class="logo">
+                <img class="logo" src="/_/img/logo-d.svg" alt="Apache Camel Logo"
+                    aria-label="white silhouette of a camel in front of a sand dune">
+            </figure>
+            <input id="footer-toggle-overview" type="checkbox" title="Show/Hide Overview section" />
             <dl>
-                <figure class="logo">
-                    <img class="logo" src="/_/img/logo-d.svg" alt="Apache Camel Logo"
-                        aria-label="white silhouette of a camel in front of a sand dune">
-                </figure>
-                <dt>Overview</dt>
-                <input type="checkbox" id="show-overview"/>
-                <label for="show-overview"><img class="show-menu" src="/_/img/show-footer-menu.svg"></label>
-                <label for="show-overview"><img class="hide-menu" src="/_/img/hide-footer-menu.svg"></label>
-                <div class="footer-menu">
-                    <dd><a href="/blog/">Blog</a></dd>
-                    <dd><a href="/docs/">Documentation</a></dd>
-                    <dd><a href="/community/support/">Community</a></dd>
-                    <dd><a href="/blog/">Download</a></dd>
-                </div>
+                <dt><label for="footer-toggle-overview">Overview</label><label for="footer-toggle-overview">&#65291;</label></dt>
```

Comment:
`<label for="footer-toggle-overview">&#65291;</label>`* is better included with a separate unique **id** to distinguish as it appears for desktop screen as well and the same case for each label with `&#65291;` in content.
- `<label for="footer-toggle-overview" id="expand">&#65291;</label>`

Reply:
Yeah I was preoccupied with mobile version that I forgot to check the desktop one. Not sure if assigning `id`s here would be the correct approach. Assigning same `id` to more than one element is not valid HTML. I've set the `display: none` via CSS selector: `footer .footer dl dt label:nth-child(2)`.